web/src/features/entitlements/constants/entitlements.ts

```diff
@@ -15,12 +15,14 @@ const entitlements = [
   "scheduled-blob-exports",
   "prompt-protected-labels",
   "admin-api",
+  "in-app-support",
 ] as const;
 export type Entitlement = (typeof entitlements)[number];
 
 const cloudAllPlansEntitlements: Entitlement[] = [
   "cloud-billing",
   "trace-deletion",
+  "in-app-support",
```

Comment:
@marcklingen: As discussed, I added the entitlement to all cloud plans. Right now this is inconsistent with https://langfuse.com/pricing